MRN-Code/coinstac
# Computation Validation

* **Problem:**
   1. Computation schemas need to be validated before submission to ensure basic adherence to the standard.
* **Solution:**
   1. Created new validator service in `coinstac-common`.
   1. Basic Pipeline and Computation Joi schemas created.
   1. Computations now run through validation prior to submission. 
* **Tests:**
   1. View _Computations_ navigation item and click Create Computation button.
   1. Add the VBM schema file from `coinstac-api-server/test/data` to see an error message complaining about the `meta.name` field.
